### ISSUE TITLE
Scope Template

### ISSUE THREAD

**Issue by 17-5** [sheagcraig/JSSImporter#73]
I was wondering if Shea or anyone else had a Scope template they were using. Running into a few issues setting the limitations properly and a little helping guide would be useful. Scripts and other templates made it a cake to add what I needed and remove what I didn't so I imagine a quick Scope template to append to the Policy Template will be the same way.

**Comment by 17-5** [comment]
And figured it out after looking at the API a bit further. Was attempting to scope to network segments.

	<scope>
    <limitations>
      <network_segments>
        <network_segment>
          <id>0</id>
          <name>Network1</name>
        </network_segment>
        <network_segment>
          <id>1</id>
          <name>Network2</name>
        </network_segment>
      </network_segments>
    </limitations>
	</scope>

**Comment by Thomgi** [comment]
Hi,

I'm trying to add exclusions on some policies using a scope template as you do for limitations, but unsuccessful.

here is what I added to my policytemplate.xml :
```
<scope>
		<exclusions>
			<computer_groups>
				<ComputerGroup>
				<id>137</id>
				<name>smartgrouptoexclude</name>
				</ComputerGroup>
			</computer_groups>
		</exclusions>
</scope>
```

instead of `<!--Scripts added by JSSImporter-->`

Where do you find API documentation ?

**Comment by jbmaruh** [comment]
Hey Thomgi,
API information is available at https://yourcasperurl.com:8443/api

What I found to be the easiest was to open up the API page and then look through the categories I was needing to scope. For instance the Network Segment I found under the Network Segments section and was laid out in the format I have above. Once you start looking at the API format a bit you'll get the handle of how it is laid out and be able to add it into JSSImporter more easily as well.

Here's a quick Exclusions template I keep just for reference for items that can be excluded.Remember that there's the Target, Limitations and Exclusions. I am generally setting my Target to <all_computers>true</all_computers> and leaving Exclusions blank. Then setting my Limitations to Smart Groups or whatever else I may need to.

`    <exclusions>
      <computers/>
      <computer_groups/>
      <buildings/>
      <departments/>
      <users/>
      <user_groups/>
      <network_segments/>
    </exclusions>`
